ops/cli.py

```diff
@@ -1,351 +1,310 @@
 #!/usr/bin/env python3
-"""WFL Deployment Script
-@rex
+"""
+WFL Deployment Script
 
-requirements: pip3 install gitpython pyyaml
+requirements: pip3 install pyyaml
 
 usage: python3 cli.py -h
 """
-from enum import Enum
-from pathlib import Path
-from typing import Callable
 import argparse
 import json
 import os
 import shutil
 import subprocess
 import sys
-import tempfile
+from dataclasses import dataclass
+from pprint import PrettyPrinter
+from typing import Dict, Callable, List
+
 import yaml
 
+from render_ctmpl import render_ctmpl
+from util.misc import info, success, error, warn, shell
+
 # In case this will need to run on Windows systems
 if sys.platform.lower() == "win32":
     os.system("color")
```

Comment:
I think we should maybe remove this because anyone running WFL stuff on Windows will need to be using some sort of Linux shell because of `make`. I use Windows for all of my work outside of Broad and I can't imagine someone would be using a terminal that wouldn't support Unix colors (all modern things do).